phpcs-boiler.xml

```diff
@@ -77,21 +79,9 @@
 		<exclude name="WordPress.Files.FileName.NotHyphenatedLowercase"/>
 	</rule>
 
-	<rule ref="WordPress.NamingConventions.PrefixAllGlobals">
-		<properties>
-			<property name="prefixes" type="array">
-				<element value="MadeWithWPPS"/>
-			</property>
-		</properties>
-	</rule>
+	<rule ref="WordPress.NamingConventions.PrefixAllGlobals"/>
 
-	<rule ref="WordPress.WP.I18n">
-		<properties>
-			<property name="text_domain" type="array">
```

Comment:
I [confirmed](https://github.com/studiopress/fse-studio/pull/270#pullrequestreview-1236592357) that this will still catch a `text_domain` error.